### ISSUE TITLE
inappropriate image src/img/af/polymc-themes.png

### ISSUE THREAD

**Issue by LennyMcLennington** [PolyMC/polymc.github.io#207]
This has just come to my attention.

This website should not contain lewd or sexual content.
This is not appropriate even if it was for a joke. You can make a joke about Google Chrome themes or whatever without it being sexual. it needs to be removed or replaced with something appropriate.

**Comment by ZekeSmith** [comment]
There was discussion around the april fools inclusion and it was also a pull request, it has been a very long time to only be complaining about this now. 

However I can agree but mainly at the fact we should probably remove the april fools page anyways or make it more clear of intentions. For some reason people think this post is actually legitimate.

**Comment by LennyMcLennington** [comment]
> There was discussion around the april fools inclusion and it was also a pull request, it has been a very long time to only be complaining about this now.

That's besides the point, if I had noticed sooner I would have made the same request that I have made just now. I even said in the start of the issue body "This has just come to my attention." specifically so you know **why** I am complaining about this only now.

> However I can agree but mainly at the fact we should probably remove the april fools page anyways or make it more clear of intentions. For some reason people think this post is actually legitimate.

I don't take issue with the april fool's page in general, just this image being used. How about just adding in big bold text at the start **DISCLAIMER: THIS POST IS A JOKE!!!! THIS POST IS NOT SERIOUS!!!!!!!!!!** then you will know anyone who still claims to think it's serious is just joking.

**Comment by ZekeSmith** [comment]
fair enough do we have an extension you would recommend to replace it with?

**Comment by ZekeSmith** [comment]
It is now kid friendly

**Comment by LennyMcLennington** [comment]
> fair enough do we have an extension you would recommend to replace it with?

Not specifically, anything that isn't inappropriate is fine.

Here's some that would be ok, but you can also choose another one if you want:

- https://chrome.google.com/webstore/detail/morbius/dpdjhgagfknncflikadmnpobabempggk
- https://chrome.google.com/webstore/detail/among-us-hd-wallpaper-the/jfmgnjgoholaaedggmkdimknoejbhmji
- https://chrome.google.com/webstore/detail/troll-theme/mhoddkiilnmbeoobemfbbgdcmngjnfha

> It is now kid friendly

The 404 thing is also fine if that's what you want.

**Comment by LennyMcLennington** [comment]
@ZekeSmith the image is under this license: https://pixabay.com/service/license/

I am not sure whether this can be used on this site. Do you know?

**Comment by DioEgizio** [comment]
🤓

**Comment by dada513** [comment]
“🤓” - 🤓

**Comment by ZekeSmith** [comment]
@LennyMcLennington it should be fine if I read the terms correctly:
![image](https://user-images.githubusercontent.com/81902668/185771894-7eaf2cb5-3f22-4aeb-ade2-fa9eafffdf21.png)
